unified-codes/libs/ui/src/lib/components/entity/molecules/EntityToggleBar.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+
+import Grid from '../../layout/atoms/Grid';
+import ToggleButton from '../../inputs/molecules/ToggleButton';
+
+import { EEntityType } from '@unified-codes/data';
+
+export interface IEntityToggleBarProps {
+  classes?: {
+    root?: string,
+    buttonContainer?: string,
+    buttonActive?: string,
+    buttonInactive?: string,
+  };
+  buttonTypes: EEntityType[],
+  buttonStates: { [key in EEntityType]: boolean },
+  buttonLabels: { [key in EEntityType]: string }
```

Comment:
Sorry to @katherine-sussol for stealing your `EntityTypeFilter`!

![mine](https://user-images.githubusercontent.com/43223637/94338467-7f657b80-0046-11eb-91ef-c00f11107740.gif)

Removed `ToggleButtonGroup` and replaced `EntityTypeFilter` with `EntityToggleBar`. WRT to the props for `EntityToggleBar`: this isn't necessarily the best way to do this! I think a probably less fussy way is to just pass: `activeTypes` and `inactiveTypes` (would be convenient for filtering also, as could just check that `activeTypes.contains(entity.type)`.

Taking it to the logical conclusion (which I think is probably the way to go, but thoughts welcomed!): I'm not sure we even need a convenience component that functions as a "mapper" for types to `ToggleButton` (just grouping the buttons is all we need)! I think the parent could do the mapping via a selector or `useMemo` and just pass an array of `ToggleButton`s directly to `EntityToggleBar`. If we go with that, all the `onClick`s would also be handled directly by the parent, e.g. defined using a dispatch function from `mapDispatchToProps` such as `(type: IEntityType) => dispatch(updateActiveTypes(type)` (the reducers handle all the messy stuff, they look at the `type` and see if it isalready enabled or not and update accordingly!) I think this is quite nice, as again it keeps all the state in the store and all the logic in the reducers!



@katherine-sussol Thoughts?